RPerformanceTracking/Private/_RPTTrackingManager.m

```diff
@@ -160,12 +160,12 @@ - (void)updateConfiguration
     NSString *thisVersion   = [thisBundle objectForInfoDictionaryKey:@"CFBundleShortVersionString"];
     
     NSBundle *appBundle     = NSBundle.mainBundle;
-    NSString *appIdentifier = appBundle.bundleIdentifier;
+    NSString *relayAppID    = [appBundle objectForInfoDictionaryKey:@"RPTRelayAppID"];
```

Comment:
This key is something set by the developer in their app's plist so we should do the same non-zero length checks that we do for the subscription key.

Reply:
Got it.. Ok

Reply:
Then, If the length is zero, we do send without app id?

Reply:
Android fails fast, because this config value is not gonna change in production apps - but it might lead to problems during integration (user forgets and gets frustrated when no data shows in the portal)
@donnie-jp what's your take on this?

Reply:
@superbgv @NemoOudeis There's an assert that fires in debug builds if that key isn't set saying `Your application's Info.plist must contain a key 'RPTRelayAppID' set to...`